--- debian/rules.orig	2021-01-17 03:38:37.237972114 +0000
+++ debian/rules	2021-01-17 03:36:50.236989864 +0000
@@ -47,6 +47,8 @@
 		--enable-auth-ntlm="fake,SMB_LM" \
 		--enable-external-acl-helpers="file_userip,kerberos_ldap_group,LDAP_group,session,SQL_session,time_quota,unix_group,wbinfo_group" \
 		--enable-security-cert-validators="fake" \
+		--enable-security-cert-generators \
+		--enable-ssl-crtd \
 		--enable-storeid-rewrite-helpers="file" \
 		--enable-url-rewrite-helpers="fake" \
 		--enable-eui \
@@ -61,7 +63,8 @@
 		--with-filedescriptors=65536 \
 		--with-large-files \
 		--with-default-user=proxy \
-		--with-gnutls
+		--with-gnutls \
+		--with-openssl
